projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -73,276 +72,293 @@ describe('DatagridComponent', () => {
             declarations: [HostWithDatagridComponent, DatagridDetailsComponent, DatagridDetailsPaneComponent],
         }).compileComponents();
 
-        this.finder = new WidgetFinder(HostWithDatagridComponent);
-        this.vcdDatagrid = this.finder.find(GridWithBoldRenderer);
-        this.clrGridWidget = this.vcdDatagrid.clrDatagrid;
-        this.component = this.finder.find(VcdDatagridWidgetObject).component as DatagridComponent<MockRecord, RecordId>;
+        this.finder = new AngularWidgetObjectFinder(HostWithDatagridComponent);
+        // @ts-ignore
+        this.vcdDatagrid = this.finder.find(VcdDatagridWidgetObject);
```

Comment:
Was this line throwing an error without the `// @ts-ignore`? If so, we might want to work together to improve the types that come out of the AngularWidgetObjectFinder. I tried to use some magic types but they might not work perfectly.

Reply:
Without the `// @ts-ignore`, there is an error:
`TS2322: Type 'CorrectReturnTypes<VcdDatagridWidgetObject<unknown>, TestElement>' is not assignable to type 'VcdDatagridWidgetObject<TestElement>'.   The types returned by 'findClrDatagrid(...)' are incompatible between these types.     Type 'CorrectReturnTypes<ClrDatagridWidgetObject<unknown>, TestElement>' is missing the following properties from type 'ClrDatagridWidgetObject<TestElement>': _getRow, _getCell`